packages/compass-sidebar/src/components/csfle-marker.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import {
+  css,
+  Icon,
+  Badge,
+  BadgeVariant,
+  useFocusRing,
+  spacing,
+} from '@mongodb-js/compass-components';
+
+const badgeContainerStyles = css({
```

Comment:
Technically the badge container and button css plus this pattern / concept of a badge that's in a button that's in the sidebar is copy/pasted but when I really started looking at extracting it it felt like overkill. Maybe something to leave until we have a third one?

Let me know where I should draw the line :)